stacks/bottlerocket-ecs-updater.yaml

```
@@ -31,6 +31,17 @@ Resources:
                 - 'ecs-tasks.amazonaws.com'
             Action:
               - 'sts:AssumeRole'
+      Policies:
+        - PolicyName: CreateLogGroupPolicy
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              # Allows creating log group if it does not exist
+              - Effect: Allow
+                Action:
+                  - 'logs:CreateLogGroup'
+                Resource:
+                  - "arn:aws:logs:*:*:*"
```

Nit: consistency

```suggestion
                  - 'arn:aws:logs:*:*:*'
```

As a side note, yaml doesn't require quotes for strings, so you could do this going forward:

```yaml
Resource:
   - arn:aws:logs:*:*:*
```

---

Done